client/src/ui/css/ResultsDisplay.css

```diff
@@ -6,57 +6,194 @@
 
 .filter-container {
   font-family: "Source Sans Pro", sans-serif;
-  padding-left: 6%;
-  padding-top: 28px;
   line-height: 37px;
-  min-width: 200px;
 }
 
-.filter-title {
-  font-family: "Source Sans Pro", sans-serif;
-  font-size: 18px;
-  font-weight: var(--semi-bold-weight);
+@media (min-width: 1000px) {
+  .filter-container {
+    padding-left: 6%;
+    padding-top: 28px;
+    min-width: 200px;
+  }
+}
+
+@media (max-width: 1000px) {
+  .filter-container {
+    margin-top: 36px;
+    margin-left: 24px;
+    margin-right: 24px;
+  }
+}
+
+.filter-title{
   font-style: normal;
-  line-height: normal;
+  color: #000000;
   align-items: center;
-  text-decoration-line: underline;
   margin-bottom: 23px;
-  color: #000000;
+}
+
+@media (min-width: 1000px) {
```

Comment:
Another thing here. I noticed that bootstrap ends the medium size at 992 pixels instead of 1000px (the effect of this is clear if you try to set width to between 992 and 1000px - the filter text size gets altered in desktop), so could you change all the 1000px to 992px in this PR? (I think you might have have it that way from the start before we decided to change it haha).

Reply:
I just did! The only problem is that the filter button is there when the width is 991. I'm not sure why exactly this happens.